test/e2e/kubevirt.go

```diff
@@ -1988,6 +2005,13 @@ ip route add %[3]s via %[4]s
 				topology: udnv1.NetworkTopologyLayer2,
 				role:     udnv1.NetworkRolePrimary,
 			}),
+			XEntry(nil, testData{
```

Comment:
Do you mind splitting the commits in a way where it is easy to revert the one skipping/modifying the tests with regards to the bug?